packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -714,12 +714,24 @@ const selfBehavior = {
     state.vaultCounter += 1;
     const vaultId = String(state.vaultCounter);
 
-    const vault = makeVault(zcf, manager, vaultId);
+    const storageNode = E(state.storageNode).getChildNode(`vault${vaultId}`);
```

Comment:
I was expecting something like `published.vaultFactory.manager6.vaults.23423`. Is there a `.vaults.` node?
Something like `published.vaultFactory.manager6.vaults.vault23423` is fine, but I wouldn't want `vaule232` to be a peer of `metrics` or `governance`.

Reply:
>  I wouldn't want vaule232 to be a peer of metrics or governance

Ah, good point. If you ask for keys of a manager you don't want every vault ever. :)